tree_diff.go

```diff
@@ -42,11 +52,27 @@ func (treediff *TreeDiff) Requires() []string {
 
 // ListConfigurationOptions returns the list of changeable public properties of this PipelineItem.
 func (treediff *TreeDiff) ListConfigurationOptions() []ConfigurationOption {
-	return []ConfigurationOption{}
+	options := [...]ConfigurationOption{{
+		Name:        ConfigTreeDiffEnableBlacklist,
+		Description: "Enable blacklisted directories.",
```

Comment:
This is a bit misleading... "Skip blacklisted directories" is better. And yep, now I see the problem, lol, please revert it back to your name `--skip-backlist`